views/js/controller/Delivery/monitoring.js

```diff
@@ -1095,10 +1095,20 @@ define([
                     if (showColumnExtendedTime) {
                         model.push({
                             id: 'extendedTime',
-                            label: __('Extended Time'),
+                            label: __('Changed Time'),
                             transform(value, row) {
-                                var extendedTimer = _.isObject(row.timer) ? row.timer : {};
-                                return (extendedTimer.extendedTime ? 'x' : '') + extendedTimer.extendedTime;
+                                let state = [];
+                                const timer = _.isObject(row.timer) ? row.timer : {};
+                                if (timer.hasOwnProperty('extendedTime') && timer.extendedTime) {
+                                    state.push(': x' + timer.extendedTime);
+                                }
+                                if (timer.hasOwnProperty('extraTime') && timer.extraTime) {
+                                    state.push(__('Extra') + ': ' + timer.extraTime);
+                                }
+                                if (timer.hasOwnProperty('adjustedTime') && timer.adjustedTime) {
+                                    state.push(__('Adjusted') + ': ' + timer.adjustedTime);
+                                }
+                                return state.join('<br />');
```

Comment:
It works, all the data is presented. 2 considerations I wanted to raise. Maybe a label in front of extended time multiplier would be useful. And extra or adjusted time values are shown as a number, in seconds. Since `Remaining` column has a time formatted in `H:i:s`, maybe here it would also be more clear like that? Maybe UI/UX people could advice better